packages/prisma/schema.prisma

```diff
@@ -417,6 +417,8 @@ model Booking {
   /// @zod.custom(imports.bookingMetadataSchema)
   metadata              Json?
   isRecorded            Boolean              @default(false)
+  iCalUID               String               @db.VarChar()
```

Comment:
@keithwillcode add `@db.VarChar()` to the schema maps the column to `VARCHAR` since Prisma will default map the column to `TEXT` for Postgres.

https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#default-type-mappings